app/views/kaminari/_prev_page.html.haml

```diff
@@ -0,0 +1,2 @@
+%li
+  = link_to_unless current_page.first?, raw(t 'views.pagination.previous'), url, :rel => 'prev', :remote => remote
```

Comment:
you have generated kaminari views. Please change the hash syntax and style it with Bootstrap.